README.md

```diff
@@ -148,8 +146,17 @@ config.server.options.config.routingOperations = ${routingOperations} [
 
 Each `handler` is specified with [Routing Handler Options](/core/docs/asciidoc/dataobjects.adoc#routinghandleroptions).
 
-#### Routing Handler
```

Comment:
We point to the "Routing Handler" section in our documentation. We should add some description here.